### PR TITLE
Fix exception not raised when unsupported format is specified when dumping computational graph

### PR DESCRIPTION
**Pull request by kmaehashi**

### CODE REVIEW

**Comment by kmaehashi** [comment]
@hvy Could you check?

File: tests/chainer_tests/test_computational_graph.py [hunk 1]
```diff
@@ -212,6 +212,9 @@ def test_dotfile_content(self):
             for key, value in style.items():
                 self.assertIn('{0}="{1}"'.format(key, value), dotfile_content)
 
+    def test_unsupported_format(self):
+        with self.assertRaises(NotImplementedError):
+            self.g.dump('graphml')
```
Comment (hvy) [inline tests/chainer_tests/test_computational_graph.py]: Could you add another empty line here (fix CI failure)?

Comment (hvy) [reply tests/chainer_tests/test_computational_graph.py]: Thanks!

**Comment by hvy** [comment]
Jenkins, test this please.

**Comment by hvy** [comment]
LGTM, CI failure is not related to this PR.

**Pull request merged**